### PR TITLE
Add VisualQnA and SearchQnA CI test

### PR DESCRIPTION
**Pull request by chensuyue**

### CODE REVIEW

File: VisualQnA/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Now you can start the frontend UI by following commands:
 ```
 cd ui/
 pip install -r requirements.txt
-python app.py --host 0.0.0.0 --port 7860 --worker-addr http://localhost:8085 --share
+http_proxy= python app.py --host 0.0.0.0 --port 7860 --worker-addr http://localhost:8085 --share
```
Comment (chensuyue) [inline VisualQnA/README.md:58]: @Spycsh missing something?

File: SearchQnA/langchain/docker/Dockerfile [hunk 2]
```diff
@@ -22,8 +22,7 @@ ENV PYTHONPATH=/home/user:/usr/lib/habanalabs/:/langchain/libs/community:/langch
 
 # Install required branch
 RUN git clone https://github.com/Spycsh/langchain.git /langchain -b master && \
-    pip install --no-cache-dir -e /langchain/libs/langchain && \
-    rm -rf /langchain
+    pip install --no-cache-dir -e /langchain/libs/langchain
```
Comment (chensuyue) [inline SearchQnA/langchain/docker/Dockerfile:25]: keep `rm -rf /langchain`, after installation the source code should be useless, right? @Spycsh

File: VisualQnA/README.md (hunk 1 above)
Comment (Spycsh) [reply VisualQnA/README.md:58]: No. It's just to unset the http_proxy for this call

File: SearchQnA/langchain/docker/Dockerfile (hunk 2 above)
Comment (Spycsh) [reply SearchQnA/langchain/docker/Dockerfile:25]: No. langchain_community is still needed

**Pull request merged**